addons/skin.estuary/xml/DialogVideoInfo.xml

```diff
@@ -225,6 +225,36 @@
 						<param name="altlabel" value="$LOCALIZE[126]: $INFO[ListItem.Status]" />
 						<param name="visible" value="!String.IsEmpty(ListItem.Status)" />
 					</include>
+					<include content="InfoDialogMetadata">
+						<param name="control_id" value="160" />
+						<param name="label" value="[COLOR button_focus]$LOCALIZE[552]: [/COLOR]$INFO[ListItem.Date]" />
+						<param name="altlabel" value="$LOCALIZE[552]: $INFO[ListItem.Date]" />
+						<param name="visible" value="!String.IsEmpty(ListItem.Date) + (String.IsEqual(ListItem.DBType,episode) | String.IsEqual(ListItem.DBType,video))" />
```

Comment:
for episodes, this will return the same as premiered.
for videos, it'll return the filedate.
not really useful, so please drop this one.

Reply:
For local content it shows when it was added on disk, which is different to when it was added to the database, or when it was last played. It has value if your content predates your Kodi installation.

The big issue is that for VOD content we are missing the right metadata, we may have:
- Date aired (transmitted on TV channel, not first aired ever like TVDB)
- Date added (to content store, not added to Kodi, semantics)
- Date removed (or time remaining)
- Date last played

So we reuse these fields.

Reply:
I have a longer list of other metadata-fields that would be useful to have in Kodi for VOD content or live streams.

Reply:
Done.